docs/commands/AfterAll.mdx

```diff
@@ -9,7 +9,6 @@ keywords:
   - Documentation
 hide_title: false
 hide_table_of_contents: false
-custom_edit_url: null
```

Comment:
this should probably remain in place?

Reply:
Good catch, just re-added the `-EditUrl` parameter back to the generate-command-reference script.